omod/src/test/resources/org/openmrs/module/fhir2/providers/ServiceRequestWebTest_create.json

```diff
@@ -0,0 +1,7 @@
+{
+  "resourceType": "ServiceRequest"
+
+
+}
+
```

Comment:
Why do you need this file with only resourceType property? It's not a valid ServiceRequest resource CREATE payload?